lib/class-wp-rest-global-styles-controller-gutenberg.php

```diff
@@ -196,28 +219,10 @@ public function get_item_permissions_check( $request ) {
 	 * @param WP_Post $post Post object.
 	 * @return bool Whether the post can be read.
 	 */
-	protected function check_read_permission( $post ) {
+	public function check_read_permission( $post ) {
```

Comment:
Done in Core in https://github.com/WordPress/wordpress-develop/commit/6fc019aa711308d6eaa1e672498b2a789a0c584c as part of inheriting `WP_REST_Posts_Controller`